lib/reek/cli/application.rb

```diff
@@ -11,53 +11,45 @@ module CLI
     # command line.
     #
     class Application
-      STATUS_SUCCESS = 0
-      STATUS_ERROR   = 1
-      STATUS_SMELLS  = 2
       attr_reader :configuration
 
       private_attr_accessor :status
       private_attr_reader :command, :options
 
       def initialize(argv)
-        @status = STATUS_SUCCESS
         @options = configure_options(argv)
+        @status = options.success_exit_code
```

Comment:
`status` now needs to be initialized after `options`, hence the moving of the lines.